hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SystemContractUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.utils;
+
+import static com.hedera.node.app.service.contract.impl.utils.ConversionUtils.tuweniToPbjBytes;
+
+import com.hedera.hapi.node.base.ContractID;
+import com.hedera.hapi.node.contract.ContractFunctionResult;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import org.apache.tuweni.bytes.Bytes;
+
+/**
+ * Utilities for system contracts.
+ */
+public class SystemContractUtils {
+
+    private SystemContractUtils() {
+        throw new UnsupportedOperationException("Utility Class");
+    }
+
+    /**
+     * Create a successful contract function result.
+     * @param gasUsed       Report the gas used.
+     * @param result        The result of the contract call.
+     * @param contractID    The contract ID.
+     * @return              The created contract function result for a successful call.
+     */
+    public static @NonNull ContractFunctionResult contractFunctionResultSuccessFor(
```

Comment:
I usually see the `@NonNull` annotation for the function result appearing _before_ the access modifiers `public` etc, did spotless leave it this way for you?

Reply:
yes it did.  I guess spotless doesn't care.  Moved